src/Device/Driver/AR62xx.cpp

```diff
@@ -195,7 +149,7 @@ class AR62xxDevice final : public AbstractDevice
    * Called by "PutActiveFrequency"
    * doing the real work
    */
-  bool AR620xPutFreqActive(double frequency, const TCHAR *stationName, OperationEnvironment &env);
+  bool AR620xPutFreqActive(double Freq, const TCHAR *StationName, OperationEnvironment &env);
```

Comment:
These new parameter names still violate the documented XCSoar naming convention.